SEPRTR/iocBoot/iocSEPRTR-IOC-01/st-daq.cmd

```diff
@@ -0,0 +1,16 @@
+## NI cDAQ-9185
+epicsEnvSet("CDAQER","cDAQ9185-1D195CFMod2")
+epicsEnvSet("CDAQAI","cDAQ9185-1D195CFMod3")
+epicsEnvSet("CDAQAO","cDAQ9185-1D195CFMod4")
```

Comment:
Are these hardware addresses which are tied to specific pieces of hardware, or are they general enough that they could be used by other instruments or institutions using the same model? If not, could they be made to be generic through macros?